src/controllers/listings.ts

```diff
@@ -28,6 +30,15 @@ export const getListings = asyncHandler(async (req, res) => {
             count: data.length,
             data,
         });
+    } else if (req.params.organisation_id) {
+        const rows = await db.manyOrNone(
+            'SELECT * FROM organisations o LEFT JOIN listingsorganisations lo ON o.organisation_id = lo.organisation_id LEFT JOIN listings l ON lo.listing_id = l.listing_id WHERE o.organisation_id = $1',
+            req.params.organisation_id,
+        );
```

Comment:
You will need to add an additional line below to remove `null` like this:
```
// remove null listing_organisation_id from rows
        const data = rows.filter((row) => row.listing_organisation_id !== null);
```

This is because a `organisations o LEFT JOIN listingsorganisations lo ON o.organisation_id = lo.organisation_id` means that for the case where `o.organisation_id` exists but `lo.organisation_id` does not, it will display the entry with `o.*` in their normal values, but `lo.*` is all `null`

You may try calling `/api/organisations/1/listings` on Postman to see how this looks like.